nft-demo/src/App.js

```diff
@@ -105,6 +106,7 @@ async function extractMeta(api) {
   if (specTypes.ExtrinsicPayloadV4) {
     delete specTypes.ExtrinsicPayloadV4;
   }
+  delete specTypes.EnhancedTokenId;
```

Comment:
what does this do?

Reply:
Need to remove all the classes/objects from the spectypes.
The reason for the removal of the following object from specTypes -
the metadata that gets updated on the polkadot extension requires types of type (Record<string, string>)
It does not allow classes/objects...

I am planning to have a different way of filtering this